scripts/yarn-install.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const { exec } = require("child_process");
```

Comment:
Destructuring won't work on Node 4.x AFAIK. Also consider using https://github.com/sindresorhus/execa instead for better cross OS support.